api.planx.uk/send/uniform.js

```diff
@@ -54,8 +59,16 @@ const sendToUniform = async (req, res, next) => {
     const { clientId, clientSecret } = getUniformClient(
       req.params.localAuthority
     );
+
     // Setup - Create the zip folder
-    const zipPath = await createZip(payload);
+    const zipPath = await createUniformSubmissionZip({
+      sessionId: payload.sessionId,
+      templateNames: payload.templateNames,
```

Comment:
passing these along in the payload seems like a fine first approach while we're still generating XML client side :ok_hand: 

one bigger picture question: do you imagine templateNames should always be stored on the `flows` table, or are these actually something we could infer from the Passport going forward? (Eg similar to how we decide to include or not boundary files, but using "application.type" = "ldc.existing" || "ldc.proposed" passport key/values). After that first passport check to determine which template(s) apply, then you could still proceed with the required fields validation and so on? 

once confirmed we can successfully send doc files to Uniform, I'm just wondering if there will be any cases where one LDC service would _not_ want to always try to generate the relevant doc(s) and if the passport-check approach would save repeated configs at the database-field level?

recognize the null-able database field is a nice proxy for a feature-flag though right now! not suggesting any changes here, just still a little unsure if templates at the flow-level make complete sense to me.

Reply:
yes totally happy for this to merge as-is right now (sorry meant for that to be clear in original comment), just meant to raise these as questions to keep in mind as we come onto those future tasks and actually hook documents up to flows/passport going forward